frontend/src/components/table/table.tsx

```
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { PropsWithChildren } from 'react';
+import { Loader } from '@/components/loader';
+import { TableProps, TableCellProps, TableRowProps } from './types';
+import './styles.css';
+
+export const TableRow = ({ children, url }: PropsWithChildren<TableRowProps>) => {
+  const Component = url ? 'a' : 'div';
+  return (
+    <Component role="row" className="wb-tableRow" {...(url ? { href: url } : {})}>
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<tr>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)